test/integration/master/kube_apiserver_test.go

```diff
@@ -138,3 +142,95 @@ func TestOpenAPIDelegationChainPlumbing(t *testing.T) {
 		t.Errorf("missing path: %q", registrationPrefix)
 	}
 }
+
+// return the unique endpoint IPs
+func getEndpointIPs(endpoints *corev1.EndpointsList) []string {
+	endpointMap := make(map[string]bool)
+	ips := make([]string, 0)
+	for _, endpoint := range endpoints.Items {
+		if endpoint.ObjectMeta.Name != "kubernetes" {
+			continue
+		}
+		for _, subset := range endpoint.Subsets {
+			for _, address := range subset.Addresses {
+				if _, ok := endpointMap[address.IP]; !ok {
+					endpointMap[address.IP] = true
+					ips = append(ips, address.IP)
+				}
+			}
+		}
+	}
+	return ips
+}
+
+func verifyEndpointsWithIPs(servers []*kubeapiservertesting.TestServer, ips []string) bool {
+	listenAddresses := make([]string, 0)
+	for _, server := range servers {
+		listenAddresses = append(listenAddresses, server.ServerOpts.GenericServerRunOptions.AdvertiseAddress.String())
+	}
+	return reflect.DeepEqual(listenAddresses, ips)
```

Comment:
Need to sort the strings list before comparing them with DeepEqual.

Reply:
Thanks! Fixed!